docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -13,78 +14,109 @@ The namespace in which to install cert-manager.
 The component always adds label `openshift.io/cluster-monitoring=true` to the namespace.
 Additionally, if component `prometheus` is installed on the cluster, the component registers the namespace to be monitored through the default Prometheus stack managed by that component.
 
-== `kubernetes_version`
+
+== `charts`
 
 [horizontal]
-type:: string
-default:: https://github.com/projectsyn/component-cert-manager/blob/master/class/defaults.yml[See `class/defaults.yml`]
+type:: object
+default::
++
+[source,yaml]
+----
+charts:
+  cert_manager:
+    source: https://charts.jetstack.io
+    version: v1.16.2
+----
 
-The Kubernetes version to provide to `helm template` when rendering the Helm chart.
+Specifies the Helm charts sources and versions for certificate-related components.
 
-TIP: Set this parameter to `${dynamic_facts:kubernetesVersion:major}.${dynamic_facts:kubernetesVersion.minor}` to use the cluster's reported Kubernetes version when rendering the Helm chart.
 
-== `charts:cert-manager`
+== `images`
 
 [horizontal]
-type:: string
-default:: see `defaults.yml`
+type:: object
+default::
++
+[source,yaml]
+----
+images:
+  cert_manager:
+    registry: quay.io
+    repository: jetstack/cert-manager-controller
+  cert_webhook:
+    registry: quay.io
+    repository: jetstack/cert-manager-webhook
+  cert_cainjector:
+    registry: quay.io
+    repository: jetstack/cert-manager-cainjector
+  cert_acmesolver:
+    registry: quay.io
+    repository: jetstack/cert-manager-acmesolver
+  cert_startupapi:
+    registry: quay.io
+    repository: jetstack/cert-manager-startupapicheck
+  kubectl:
+    registry: quay.io
+    repository: appuio/oc
+    tag: 'v4.16'
+----
 
-The version of the Helm chart to use.
+Image registry configurations for components.
 
-== `letsencrypt_email`
 
-[horizontal]
-type:: string
-default:: ``
+== `config`
 
-The email address Let's Encrypt will use to contact you.
+Configuration options for certificate-related components.
 
-== `dns01-recursive-nameservers`
+=== `config.common`
 
 [horizontal]
-type:: string
-default:: `1.1.1.1:53`
+type:: object
+default::
++
+[source,yaml]
+----
+config:
+  common:
+    httpProxy: ''
+    httpsProxy: ''
+    noProxy: ''
+    email: ''
```

Comment:
Think that's not in `common`? But still root level?

Reply:
I can move back to `helm_values` or `helmValues` 🤔 For legacy reasons probably more like helmValues, so I can work with deprecated configs in `helm_values` (bc i need the hierarchy for integrating the exoscale webhook).
Same for `config.cert_manager`, hierarchy for integrating exoscale webhook.

Reply:
Oh, I can move it to `component.cert_manager`, that makes more sense I guess.

Reply:
Sounds good! The less levels in yaml the less confusing it is 😅

No strong opinions if `helmValues` or `helm_values` we have a wide mix in most components 🙈